units/initrd-udevadm-cleanup-db.service.in

```diff
@@ -11,7 +11,7 @@
 Description=Cleanup udevd DB
 DefaultDependencies=no
 ConditionPathExists=/etc/initrd-release
-Conflicts=systemd-udevd.service systemd-udevd-control.socket systemd-udevd-kernel.socket
+Conflicts=systemd-udevd.service systemd-udev-trigger.service systemd-udevd-control.socket systemd-udevd-kernel.socket
```

Comment:
Looks reasonable, but `After=` needs the same addition. Otherwise, both services could run in parallel for a bit, and we don't want that.

Reply:
@keszybz agree, also I need conflicts/after on system-udev-settle.service, if it ever is used in both initrd & system, it will have the same bug.